src/test/java/teammates/test/cases/ui/browsertests/AdminEmailPageUiTest.java

```diff
@@ -66,22 +79,169 @@ private void testCompose() {
         emailPage.inputSubject("Email Subject");
         emailPage.inputContent("Email to save");
         emailPage.clickSaveButton();
-        emailPage.verifyHtmlMainContent("/adminEmailSaveSuccess.html");
+        assertTrue(hasStatusMessageSaveSuccess());
     }
 
     private void testSent() {
         emailPage.clickSentTab();
-        emailPage.verifyHtmlMainContent("/adminEmailSentPage.html");
+        assertTrue(isEmailSentDataDisplayCorrect());
     }
 
     private void testDraft() {
         emailPage.clickDraftTab();
-        emailPage.verifyHtmlMainContent("/adminEmailDraftPage.html");
+        assertTrue(isEmailDraftDataDisplayCorrect());
     }
 
     private void testTrash() {
         emailPage.clickTrashTab();
-        emailPage.verifyHtmlMainContent("/adminEmailTrashPage.html");
+        assertTrue(isEmailTrashDataDisplayCorrect());
+    }
+    
+    private boolean isEmailComposePageCorrect() {
+        return emailPage.isElementPresent(By.id("addressReceiverEmails"))
+            && emailPage.isElementPresent(By.name("emailsubject"))
+            && emailPage.isElementPresent(By.className("mce-tinymce"))
+            && emailPage.isElementPresent(By.id("adminEmailBox"))
+            && emailPage.isElementPresent(By.id("composeSubmitButton"))
+            && emailPage.isElementPresent(By.id("composeSaveButton"));
+    }
+    
+    private boolean hasStatusMessageNoRecipient() {
+        return emailPage.getStatus().equals("Error : No reciver address or file given");
+    }
+    
+    private boolean hasStatusMessageRecipientEmailFormatError() {
+        return emailPage.getStatus().contains("\"recipient\" is not acceptable to TEAMMATES as an email "
+                                            + "because it is not in the correct format. An email address "
+                                            + "contains some text followed by one '@' sign followed by "
+                                            + "some more text. It cannot be longer than 45 characters. "
+                                            + "It cannot be empty and it cannot have spaces.");
+    }
+    
+    private boolean hasStatusMessageNoSubject() {
+        return emailPage.getStatus().equals("\"\" is not acceptable to TEAMMATES as email subject because "
+                                          + "it is empty. The value of email subject should be no longer "
+                                          + "than 200 characters. It should not be empty.");
+    }
+    
+    private boolean hasNoErrorMessage() {
+        return !(emailPage.isElementPresent(By.className("alert-danger")));
+    }
```

Comment:
Should be better making this `hasErrorMessage()` and negating it when necessary instead.